buildlib/tools/coverity.sh

```diff
@@ -79,5 +92,14 @@ run_coverity() {
 	return $rc
 }
 
-prepare_build
+az_init_modules
+modules_for_coverity
```

Comment:
seems like you can still keep it inside `run_coverity`

Reply:
i think configure step needs it because it load gdrcopy headers etc.